---
online: 1   # 数据来源于串口设置为1，数据来源于文件设置为0

## 数据来源于串口
input_serial_port: "/dev/ttyACM0"
serial_baud_rate: 921600

input_rtcm: 0   # 是否将RTCM数据输入接收机用于RTK解算
rtcm_tcp_port: 3503

config_receiver_at_start: 0   # 是否在启动时配置接收机 ** 该功能目前暂时不支持 **
receiver_config_filepath: "~/ublox_driver_ros2/config/ublox_receiver_config.yaml"

## 数据来源于文件
ubx_filepath: "~/ublox_driver_ros2/data_resource/data_input/example.ubx"
rtk_correction_ecef:   # 当基站的位置在 RTCM 数据中报告存在偏差时，系统会通过 RTK 修正来纠正偏差
    rows: 3            
    cols: 1
    dt: d
    data: [ 0, 0, 0 ]

## 数据输出形式
to_ros: 1   # 是否发布为ros话题

to_file: 0   # 是否将数据存储在文件中
dump_dir: "~/ublox_driver_ros2/data_resource/data_output/"   # 指定数据存储目录

to_serial: 0   # 是否将数据发布到串口用于调试
output_serial_port: "/dev/ttyACM0"   # 指定数据发布串口